uis/layover/app/partials/group-edit.html

```diff
@@ -41,6 +41,11 @@
         <label>Enabled?</label>
         <span class="hint" ng-if="!raw.enabled && raw.disableExpiration">Set to automatically re-enable at {{raw.disableExpiration | dateFormat}}</span>
       </div>
+      <div class="detail-field">
```

Comment:
I'm not sure it makes sense to have authoritative indexing for a group...or rather, it might be implied that a group uses authoritative indexing if all of its member repositories do. But even then, we'll have to go through a lot more work to maintain group-level index entries for authoritative group indexing...adding as well as deleting any time a member is added.

IMO we should leave that option commented out for groups for now...